src/main/java/org/mule/extension/email/internal/mailbox/imap/IMAPOperations.java

```diff
@@ -83,13 +85,15 @@
                                                                                                          defaultValue = DEFAULT_PAGE_SIZE) int pageSize,
                                                                                                      @Optional(
                                                                                                          defaultValue = UNLIMITED) int limit,
-                                                                                                     StreamingHelper streamingHelper) {
+                                                                                                     StreamingHelper streamingHelper,
+                                                                                                     @ParameterGroup(
```

Comment:
Is this parameter group being created to avoid duplication of the parameter definition?